src/mlpack/tests/softmax_regression_test.cpp

```diff
@@ -663,7 +664,8 @@ BOOST_AUTO_TEST_CASE(SoftmaxRegressionComputeProbabilitiesAndLabelsTest)
 
   for (size_t i = 0; i < data.n_cols; ++i)
   {
-    BOOST_REQUIRE_CLOSE(arma::sum(probabilities.col(i)), 1.0, 1e-5);
+    double value = arma::sum(probabilities.col(i));
+    BOOST_REQUIRE_CLOSE(std::isnan(value)?1.0:value, 1.0, 1e-5);
```

Comment:
Hm, not sure I have seen any tests failures in the past, where this could be a problem, did the test fail on your system?

Reply:
yes, after debugging I saw the value of this sum comes nan so added this for being safe side.